src/main/java/net/sf/jabref/groups/AutoGroupDialog.java

```diff
@@ -96,7 +96,7 @@ public void actionPerformed(ActionEvent e) {
                         new ExplicitGroup(Localization.lang("Automatically created groups"),
                                 GroupHierarchyType.INCLUDING));
                 Set<String> hs = null;
-                String field = field();
+                String field1 = field();
```

Comment:
Maybe renaming the variable to "fieldText" would be more intuitive. 

Side note: Why the hell has the method "field()" been introduced, which does nothing more than returning `field.getText()`? 

@oscargus Would you remove those rather useless methods (`field(), remove()`), too? Thanks! :wink: